tla/consistency/Consistency.tla

```diff
@@ -0,0 +1,361 @@
+
+
+DO NOT MODIFY THIS FILE!!! MODIFY THE INDIVIDUAL MODULES FROM WHICH THIS IS MANUALLY STICHED TOGETHER!!!
```

Comment:
Does this need to be manually stitched? Can it not be scripted? Our experience with checking in generated files is uniformly poor, automation proves preferable every time in the long run.

Reply:
No, I'm not going to write a script.

Reply:
https://github.com/will62794/tla-web/issues/14